packages/components/src/components/card/card.tsx

```diff
@@ -86,7 +86,7 @@ export class Card {
   render() {
     return (
       <Host>
-        <div class={
+        <div aria-labelledby="label" class={
```

Comment:
label is not a sufficient description here. Guess you could access the overline or headline here. Or simply name the label card with the direction and if it has an image or not